### ISSUE TITLE
Need modifications in code

### ISSUE THREAD

**Issue by DivyaIraTech** [ravi-prakash-iratech/JavaCrudOperations#1]
1. Please use business logic in service class not in controller
2. Use lombok getter and setter annotations instead of using getter and setter method in all places 
3. Throw exceptions also in service class 
4. Don't send the Internal server error HTTP status in any case as it suggests that code is not working properly instead of use Bad Request or anything

**Comment by ravi-prakash-iratech** [comment]
Got it.Thanks

**Comment by ravi-prakash-iratech** [comment]
Done!.Please review

**Comment by DivyaIraTech** [comment]
Yes, everything is ok but no need to make constructor or toString() method, you can use annotations instead of like @Data, @AllArgsConstructor, @NoArgsConstructor
